inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/sink/mysql/MySQLSinkDTO.java

```diff
@@ -221,7 +221,7 @@ public static String filterSensitive(String url) {
             return url;
         }
         try {
-            String resultUrl = url;
+            String resultUrl = url.replaceAll(InlongConstants.BLANK, "");
```

Comment:
Please check other urls at the same time, thanks.

Reply:
@fuweng11 I think we should replace/trim the data when it comes in.
Because the external data is untrustworthy, it is processed uniformly when it enters the system;

Once the data is saved in the DB of our system, the data transferring within the system should be regarded as credible data and does not need to be processed repeatedly in every place of use.

Reply:
done